src/utils/prettifyDate.ts

```diff
@@ -1,5 +1,15 @@
-export const prettifyDate = (dateEpoch: number, options?: Record<string, unknown>): string => {
-  const params = options || { dateStyle: "long" }
+import { LANGUAGE_CODES } from "../app.constants";
+
+export const prettifyDate = ({ 
+    dateEpoch, 
+    locale, 
+    options = { dateStyle: "long" }
+  }: {
+    dateEpoch: number,
+    locale?: string,
+    options?: Record<string, unknown>
```

Comment:
We can use Intl.DateTimeFormatOptions instead to make it more strict

Reply:
Nice suggestion, have done!